src/packagedcode/rubygems.py

```diff
@@ -120,7 +122,8 @@ def recognize(cls, location):
 
         if location.endswith('.gemspec'):
             # TODO: implement me
```

Comment:
Remove this TODO, since I believe this is what you are currently doing.

Reply:
Yes. It is corrected now :)